projects/api/prisma/schema.prisma

```diff
@@ -19,33 +30,174 @@ model Country {
+  createdAt DateTime  @default(now()) @map(name: "created_at") @db.Timestamptz()
+  updatedAt DateTime  @default(now()) @updatedAt @map(name: "updated_at") @db.Timestamptz()
+  products  Product[]
 
   @@map(name: "brands")
 }
 
 model Supplier {
-  id           String   @id @default(uuid()) @db.Uuid()
-  email        String   @unique @db.Citext
-  deliveryTime Int      @map(name: "delivery_time")
-  createdAt    DateTime @default(now()) @map(name: "created_at") @db.Timestamptz()
-  updatedAt    DateTime @default(now()) @updatedAt @map(name: "updated_at") @db.Timestamptz()
-  countryId    String   @map(name: "country_id") @db.Uuid()
-  country      Country  @relation(fields: [countryId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  id                String              @id @default(uuid()) @db.Uuid()
+  email             String              @unique @db.Citext
+  deliveryTime      Int                 @map(name: "delivery_time")
+  createdAt         DateTime            @default(now()) @map(name: "created_at") @db.Timestamptz()
+  updatedAt         DateTime            @default(now()) @updatedAt @map(name: "updated_at") @db.Timestamptz()
+  countryId         String              @map(name: "country_id") @db.Uuid()
+  country           Country             @relation(fields: [countryId], references: [id], onDelete: Restrict, onUpdate: Cascade)
+  productsSuppliers ProductsSuppliers[]
 
   @@map(name: "suppliers")
 }
+
+model User {
+  id            String          @id @default(uuid()) @db.Uuid()
+  name          String
+  lastName      String          @map(name: "last_name")
+  email         String          @unique @db.Citext
+  password      String
+  role          Role
+  createdAt     DateTime        @default(now()) @map(name: "created_at") @db.Timestamptz()
+  updatedAt     DateTime        @default(now()) @updatedAt @map(name: "updated_at") @db.Timestamptz()
+  penalty       Penalty?
+  wishList      WishList?
+  purchases     Purchase[]
+  usersProducts UsersProducts[]
+
+  @@map(name: "users")
+}
+
+model Penalty {
+  id        String   @id @default(uuid()) @db.Uuid()
+  amount    Int      @default(0)
+  createdAt DateTime @default(now()) @map(name: "created_at") @db.Timestamptz()
+  updatedAt DateTime @default(now()) @updatedAt @map(name: "updated_at") @db.Timestamptz()
+  userId    String   @unique @map(name: "user_id") @db.Uuid()
+  user      User     @relation(fields: [userId], references: [id], onDelete: Restrict, onUpdate: Cascade)
+
+  @@map(name: "penalties")
+}
+
+model WishList {
+  id                String              @id @default(uuid()) @db.Uuid()
+  title             String              @default("My WishList")
+  description       String              @default("")
+  createdAt         DateTime            @default(now()) @map(name: "created_at") @db.Timestamptz()
+  updatedAt         DateTime            @default(now()) @updatedAt @map(name: "updated_at") @db.Timestamptz()
```

Comment:
Why does the updatedAt have a value by default if it is not updated at the first time?

Reply:
we will fix that in a moment